bazaarvoice/emodb
# EMO-7305 update/add blob s3 configuration

## Github Issue #

-

## What Are We Doing Here?
 - add additional blob s3 configuration options
 - add s3 client rate limiting
 - refactoring

## How to Test and Verify

1. Check out this PR
2. `mvn clean verify`
3. Run gatekeeper against
4. Profit

## Risk

### Level 

`Low`
### Required Testing

`Manual`

### Risk Summary

Add one or a few complete sentences about the possible risks or concerns for
this change.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
